### PR TITLE
Integrate redux for StaticAppBar

### LINKED ISSUE #1972: Integrate redux for StaticAppBar
**Actual Behaviour**
Not present

**Expected Behaviour**
Integrate redux to improve and standardize the repo.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #1972 

Changes: Integrated both the API's in StaticAppBar with redux
Surge Deployment Link: https://pr-1973-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 Please review.

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd I have removed the initialiseshowAdmin call completely as it was not being used anywhere. api getAdmin() takes care of everything. showAdmin is the property of object (payload) returned by getAdmin(), which is being used everywhere in the code.

**Comment by harsh-jindal** [comment]
@fragm3 Please Review.

File: src/api/index.js [hunk 1]
```diff
@@ -275,6 +275,13 @@ export function fetchConverstionResponse(payload) {
   });
 }
 
+// StaticAppBar API
```
Comment (fragm3) [inline src/api/index.js]: Remove comment

File: src/redux/reducers/app.js [hunk 2]
```diff
@@ -76,6 +78,20 @@ export default handleActions(
         userSkills,
       };
     },
+    [actionTypes.APP_LIST_USER_SETTINGS](state, { payload }) {
+      let userName = '';
+      if (payload.settings && payload.settings.userName) {
+        userName = payload.settings.userName;
+      }
+      cookies.set('username', userName, {
```
Comment (fragm3) [inline src/redux/reducers/app.js]: Let's keep reducer for modifying state only, move this to local component

Comment (harsh-jindal) [reply src/redux/reducers/app.js]: @fragm3 Okay.

Comment (harsh-jindal) [reply src/redux/reducers/app.js]: @fragm3 Shall I keep an action associated with it or directly make an api call?

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes, please review.

File: src/index.js [hunk 3]
```diff
@@ -85,6 +90,21 @@ class App extends React.Component {
...
+          if (payload.settings && payload.settings.userName) {
+            userName = payload.settings.userName;
+          }
+          cookies.set('username', userName, {
+            path: '/',
+            domain: cookieDomain,
+          });
+        })
+        .catch(error => {
+          console.log(error);
```
Comment (akshatnitd) [inline src/index.js]: Remove `console.log`

**Comment by harsh-jindal** [comment]
@akshatnitd Removed `console.log` . Please review

**Comment by harsh-jindal** [comment]
@akshatnitd Sure, will keep that in mind from next time.

**Pull request merged**